### PR TITLE
libzip: update to 1.10.0

### PR DESCRIPTION
**Pull request by scribam**

### CODE REVIEW

File: mingw-w64-libzip/PKGBUILD [hunk 1]
```diff
@@ -23,7 +23,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
              "${MINGW_PACKAGE_PREFIX}-cc")
 options=('staticlibs' '!libtool')
```
Comment (Biswa96) [inline mingw-w64-libzip/PKGBUILD:24]:
```suggestion
```

**Comment by Biswa96** [comment]
man files are not present in new package.

**Comment by scribam** [comment]
man files are now correctly generated with `groff` added in `makedepends`

**Comment by Biswa96** [comment]
Thank you.

**Pull request merged**